From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Mario Carrillo <mario.alfredo.c.arevalo@intel.com>
Date: Thu, 7 Apr 2016 17:32:12 -0500
Subject: [PATCH] change include dir path

This patch was taken from
http://pkgs.fedoraproject.org/cgit/rpms/dialog.git/tree/ however
some position lines did not match, for that reason was rewritten.

Signed-off-by: Mario Carrillo <mario.alfredo.c.arevalo@intel.com>
Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 dialog-config.in | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/dialog-config.in b/dialog-config.in
index 69064e6..7ac23e2 100644
--- a/dialog-config.in
+++ b/dialog-config.in
@@ -55,6 +55,7 @@ while test $# -gt 0; do
 	eval libdir='"@libdir@"'
 	eval datadir='"@datadir@"'
 	eval mandir='"@mandir@"'
+	eval includedir='"@includedir@"'
 
 	case "$1" in
 	-*=*)
@@ -84,10 +85,7 @@ while test $# -gt 0; do
 		;;
 	# compile/link
 	--cflags|--cflags-only-I)
-		INCS=
-		if test "${prefix}/include" != /usr/include ; then
-			INCS="-I${prefix}/include"
-		fi
+		INCS="-I$includedir"
 		sed -e 's,^[ ]*,,' -e 's, [ ]*, ,g' -e 's,[ ]*$,,' <<-ENDECHO
 			$INCS
 ENDECHO
